.github/workflows/auto-update-labels.yaml

```diff
@@ -0,0 +1,32 @@
+name: Auto-update labels
+on:
+  push:
+    paths:
+      - 'misc/triage/labels.yaml'
+    branches:
+      - main
+
+jobs:
+  deploy:
+    name: Auto-update labels
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout main
+        uses: actions/checkout@v3.6.0
+        with:
+          fetch-depth: 0
```

Comment:
Do we need all hostory?

Reply:
Right!
Changed in https://github.com/aquasecurity/trivy/pull/5200/commits/d901935b80dbaaf6546ea2cee6cfe181441fc901